dodola/services.py

```diff
@@ -641,21 +642,24 @@ def correct_wet_day_frequency(x, out, process):
 
 
 @log_service
-def correct_small_dtr(x, out, threshold=1.0):
-    """Corrects small diurnal temperature range (DTR) values in a dataset
+def correct_dtr(x, out, floor=1.0, ceiling=70.0):
+    """Corrects diurnal temperature range (DTR) values in a dataset
```

Comment:
We need to describe what's actually happening aside from a "correction". It's a bit more complex now. At least outline the main components.

This is a good place for a one-line summary at the top of the docstr and then an extended description below it:

```python
""""This is a one line-summary of what the function does.

This is an extended description talking about things beyond one line and going into
gross detail about all the methods and details and all that stuff. If we needed more 
than one line for the summary description, its usually a sign that our function was trying 
to do too much.

Parameters
-----------
...
"""
```

I can live without extensive documentation on the CLI but we need to cover ouselves better in `dodola.services` and `dodola.core`. I feel like we're going to come back to this in a month with a poor idea about what all this actually does.

Reply:
Yes that totally makes sense. I wrote this too fast 👍